robottelo/constants/__init__.py

```diff
@@ -565,6 +566,15 @@
         'distro': DISTRO_RHEL8,
         'key': 'rhst',
     },
+    'rhel8_bos_ks': {
+        'id': 'rhel-8-for-x86_64-baseos-kickstart',
+        'name': 'Red Hat Enterprise Linux 8 for x86_64 - BaseOS (Kickstart)',
```

Comment:
```suggestion
        'name': 'Red Hat Enterprise Linux 8 for x86_64 - BaseOS Kickstart 8.4',
```

There's multiple version so the latest one is 8.4 which changes the base name once selected.

Reply:
@latran How were we supposed to figure out the syntax of the repo name? That was causing problems, and I was hoping there was somewhere that listed reposets and their repos.

Reply:
Enabling the repo on the UI shows the name.  You can also check the name when you use hammer command to get the repo info and that will show you the name as well.